src/main.ts

```diff
@@ -56,98 +57,92 @@ const client = new Client({
   ]
 })
 
+async function fetchActiveThreads(forumChannel: ForumChannel): Promise<Array<HelpThread>> {
+  const helpThreads = new Array<HelpThread>()
+  try {
+    const fetchedThreads = await forumChannel.threads.fetchActive()
```

Comment:
For consistency with the archived case.

```suggestion
    const fetched = await forumChannel.threads.fetchActive()
```